content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +661,76 @@ of single-resource API requests, then aggregates the responses if needed.
 By contrast, the Kubernetes API verbs **list** and **watch** allow getting multiple
 resources, and **deletecollection** allows deleting multiple resources.
 
+## Field validation
+
+  {{< feature-state for_k8s_version="v1.25" state="beta" >}}
+
```

Comment:
We should describe more about the other forms of field validation (not related to this feature being documented).

Something like:
```markdown
Kubernetes always validates the type of fields. For example, if a field in the API is defined
as a number, you cannot set that field to a text value. If a field is defined as an array of strings,
you can only provide an array. Some fields allow you to omit them, other fields are required.
Omitting a required field from an API request is an error.

If you make a request with an extra field, one that the cluster's control plane does not recognize,
then the behaviour of the API server is more complicated.
```

What do you think?

Reply:
yep, that sounds good to me. I agree it seems weird to jump right into the unknown field bits when there's more to validation than just that.